scripts/release/release-notes-onprem.sh

```diff
@@ -144,7 +161,15 @@ echo "Retreiving related tickets..."
 
 echo "checking the diff between tags $PROD_TAG and $DEV_TAG"
 
-for i in $(git log $PROD_TAG...$DEV_TAG | grep -Eo '(PACT-|CC-)([0-9]+)' | sort | uniq); do
+
+if [ -f "jira-ids.txt" ]; then
```

Comment:
if we place a jira-ids.txt in the pactflow-application folder it will use that to run script against

Reply:
this is fine, but I don't fully understand the use case?

Reply:
this I added, when we had a huge list to manually review, I think in those cases this would be useful to review and then generate the changes